frontends/verific/verific.cc

```diff
@@ -1789,8 +1789,10 @@ struct VerificExtNets
 				new_net = new Net(name.c_str());
 				nl->Add(new_net);
 
+			#ifndef NDEBUG
 				Net *n = route_up(new_net, port->IsOutput(), ca_nl, ca_net);
```

Comment:
It really looks like this call should have side effects -- are you sure it is meant to be inside `NDEBUG`?

Reply:
Oooops. Fixed in 27d59dc.